cmd/createcluster.go

```diff
@@ -121,6 +125,28 @@ func runCreateCluster(ctx context.Context, w io.Writer, conf clusterConfig) erro
 		return errors.Wrap(err, "mkdir")
 	}
 
+	if conf.DefFile != "" {
+		def, err := loadDefinition(ctx, conf.DefFile)
+		if err != nil {
+			return err
+		}
+		def.DefinitionHash = nil
```

Comment:
Either clear both operators and definition hash, or clear neither. I would say a more conservative approach is better by expecting valid data that doesn't need to be fixed.

Reply:
I'm leaning towards clearing both. Reason being that we need to clear `DefinitionHash` since a serialized definition file always contains a `definition-hash`.

Reply:
Cool, let's not clear neither since an invalid "partial" definition will be flagged by the `VerifyPartialHashes()` function